homeassistant/components/amberelectric/translations/en.json

```diff
@@ -0,0 +1,27 @@
+{
+    "title": "amberelectric",
+    "config": {
+        "step": {
+            "user": {
+                "data": {
+                    "api_token": "API Token",
+                    "site_id": "Site ID",
+                    "site_nmi": "Site NMI",
+                    "site_name": "Site Name"
+                },
+                "title": "Amber Electric",
+                "description": "Go to https://app.amber.com.au/developers to generate an API key"
+            },
+            "site": {
+                "data": {
+                    "api_token": "API Token",
+                    "site_id": "Site ID",
+                    "site_nmi": "Site NMI",
+                    "site_name": "Site Name"
+                },
+                "title": "Amber Electric",
+                "description": "Select the NMI of the site you would like to add"
```

Comment:
Why does the user needs to know the NMI? Why can't we show names ?

Reply:
That's all that is in the API at this time - We have plans to add nick names for properties, but it's not implemented yet. We don't expose the address via the API for privacy reasons.